### DISCUSSION TITLE
Outdated [@types/cytoscape] types

### DISCUSSION THREAD

**Discussion by ghost** [DefinitelyTyped/DefinitelyTyped#58920] [Issues with a @types package]
`BreadthFirstLayoutOptions` (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cytoscape/index.d.ts#L5268)

should have `maximalAdjustments` be renamed to `maximal`.

https://js.cytoscape.org/#layouts/breadthfirst

Also shouldn't the rest of the properties besides `name` be optional?

[a comment by an automated account is not shown]

**Comment by wy193777** [comment]
You can submit a pull request directly.
